card/mod/core/spec/set/all/rename_spec.rb

```diff
@@ -159,6 +159,12 @@ def assert_rename card, new_name
       expect_content_of("blue linker 2").to eq "[[Red]]"
     end
 
+    it "handles link to and nest of same card" do
```

Comment:
this test fails without the new "uniq" handling in references.  This is why I couldn't rename "Inclusion Syntax" on decko.org.